Problem Statement 1:
    
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on
blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet
have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an
effect or not.

H0 : mean = 100

H1 : mean > 100

population mean = 100

population SD = 15

n = 36

sample mean = 108

Z(test) = ((Sample mean - population mean) / (population SD/sqrt(n))) = 3.2

Let alpha = 0.05

Z(critical) = 1.64 (since it is right tailed test)

Since Z(test) > Z(critical) it falls into the critical region, hence null hypothesis is rejected.

Hence cornstarch has an effect

-------------------------------------

Problem Statement 2:
    
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second
state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple
random sample of 100 voters are surveyed from each state.
What is the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state?

P1 = 0.52

P2 = 0.47

p1 = 0.48

p2 = 0.53

n1 = 100

n2 = 100

n1P1 = 100*0.52 = 52

n1p1 = 100*0.48 = 48

n2P2 = 100*0.47 = 47

n2p2 = 100*0.53 = 53

Sample size is large enough

P1 - P2 = 0.05

SD = 0.0706

Z = -0.7082

we find that the probability of a z-score being -0.7082 or less is 0.24.

Therefore, the probability that the survey will show a greater percentage of 

Republican voters in the second state than in the first state is 0.24.

--------------------------------


Problem Statement 3:
    
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

In [5]:
z = ((1100 - 1026) / 209)
print(round(z,3))

0.354


Z = 0.354, meaning I am 0.354 Standard deviation above the mean.

I am at 63.68th  percentile, which is fairly good.